packages/core/mesh/messaging/src/messenger.ts

```diff
@@ -133,6 +144,31 @@ export class Messenger {
     return promise;
   }
 
+  private async errorLimitCheck() {
+    log(`checking error limit ${this._errorCount}`);
+    if (this._errorCount++ > ERROR_LIMIT) {
+      this.rebootNetwork();
+    }
+  }
+
+  private async rebootNetwork() {
```

Comment:
add a todo to move this into signal client and possible make it react to browser events (on sleep?) instead of errors